### PR TITLE
Add Hazelcast Jet Helm Chart

### PR DESCRIPTION
**Pull request by eminn**
### What this PR does / why we need it:
Add Helm Chart for Hazelcast Jet (https://jet.hazelcast.org/)

#### Which issue this PR fixes
N/A

#### Checklist
This is an initial PR for Hazelcast Jet, which will be maintained by the Hazelcast Organization (exact names are mentioned in Charts.yaml and OWNERS).

### CODE REVIEW

**Comment by leszko** [comment]
@davidkarlsen , could you kindly have a look at this PR? We belive it's ready to be merged.

File: stable/hazelcast-jet/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+| `rbac.create`                              | Enable installing RBAC Role authorization                                                                      | `true`                                               |
+| `serviceAccount.create`                    | Enable installing Service Account                                                                              | `true`                                               |
+| `serviceAccount.name`                      | Name of Service Account, if not set, the name is generated using the fullname template                         | `nil`                                                |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
+  --set cluster.memberCount=3,serviceAccount.create=false \
+    hazelcast/hazelcast-jet
```
Comment (davidkarlsen) [inline stable/hazelcast-jet/README.md]:
```suggestion
    stable/hazelcast-jet
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**